Jellyfin.Server.Implementations/Metrics/SessionMetrics.cs

```diff
@@ -0,0 +1,184 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using MediaBrowser.Controller.Library;
+using MediaBrowser.Controller.Session;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using Prometheus;
+
+namespace Jellyfin.Server.Implementations.Metrics.SessionMetrics
+{
+    /// <summary>
+    /// Provides Prometheus metrics related to Jellyfin sessions.
+    /// </summary>
+    public class SessionMetrics : IHostedService
+    {
+        private readonly ILogger<SessionMetrics> _logger;
+        private readonly ISessionManager _sessionManager;
+
+        private static readonly Gauge _activeSessions = Prometheus.Metrics
+            .CreateGauge("jellyfin_sessions_active_total", "Number of currently active sessions");
+
+        private static readonly Gauge _totalSessions = Prometheus.Metrics
+            .CreateGauge("jellyfin_sessions_total", "Total number of sessions, active and inactive");
```

Comment:
I don't think there's a need for two different metric names when you can consolidate this into `jellyfin_sessions_total` with a `state={active,inactive}` label.